src/engine/ruleset/integrations/windows/rules/defense-evasion-timestomp-sysmon.yml

```diff
@@ -0,0 +1,53 @@
+name: rule/defense-evasion-timestomp-sysmon/0
+
+metadata:
+  module: Windows
+  title: File Creation Time Changed
+  description: Identifies modification of a file creation time. Adversaries may modify file time attributes to blend
+    malicious content with existing files. Timestomping is a technique that modifies the timestamps of
+    a file often to mimic files that are in trusted directories.
+  versions: [Vista, "7", "8", "10", "11", Server 2012, Server 2016, Server 2019, Server 2022]
+  compatibility: This rule has been tested on Wazuh version 4.5.
+  author:
+    name: Wazuh, Inc.
+    date: 2023/09/15
+
+check: >-
+    array_contains($event.category, file) AND array_contains($event.type, change) AND $event.id == "2"
+    AND NOT regex_match($process.executable, '([A-Za-z]:)?\\\\Program Files\\\\([A-Za-z])')
+    AND NOT regex_match($process.executable, '([A-Za-z]:)?\\\\Program Files (x86)\\\\([A-Za-z])')
+    AND NOT regex_match($process.executable, '([A-Za-z]:)?\\\\Windows\\\\system32\\\\msiexec\.exe')
+    AND NOT regex_match($process.executable, '([A-Za-z]:)?\\\\Windows\\\\syswow64\\\\msiexec\.exe')
+    AND NOT regex_match($process.executable, '([A-Za-z]:)?\\\\Windows\\\\syswow64\\\\svchost\.exe')
+    AND NOT regex_match($process.executable, '([A-Za-z]:)?\\\\WINDOWS\\\\syswow64\\\\backgroundTaskHost\.exe')
+    AND NOT regex_match($process.executable, '([A-Za-z]:)?\\\\Users\\\\[A-Za-z]+\\\\AppData\\\\Local\\\\Google\\\\Chrome\\\\Application\\\\chrome\.exe')
+    AND NOT regex_match($process.executable, '([A-Za-z]:)?\\\\Users\\\\[A-Za-z]+\\\\AppData\\\\Local\\\\slack\\\\app-[A-Za-z]+\\\\slack\.exe')
+    AND NOT regex_match($process.executable, '([A-Za-z]:)?\\\\Users\\\\[A-Za-z]+\\\\AppData\\\\Local\\\\GitHubDesktop\\\\app-[A-Za-z]+\\\\GitHubDesktop\.exe')
+    AND NOT regex_match($process.executable, '([A-Za-z]:)?\\\\Users\\\\[A-Za-z]+\\\\AppData\\\\Local\\\\Microsoft\\\\Teams\\\\current\\\\Teams\.exe')
+    AND NOT regex_match($process.executable, '([A-Za-z]:)?\\\\Users\\\\[A-Za-z]+\\\\AppData\\\\Local\\\\Microsoft\\\\OneDrive\\\\OneDrive\.exe')
```

Comment:
Should we include the special character ^ at the beginning of the regular expressions so that the path matches at the beginning and not anywhere in the string?

Reply:
no since it is optional in this case